src/components/dashboard/UserChoiceDashboard.js

```diff
@@ -16,6 +16,10 @@ export const UserChoiceDashboard = () => {
     const { t } = useTranslation();
     const darkModeStatus = useRecoilValue(darkModeState);
 
+    function truncateContent(content, maxLength = 20) {
+        return content.substring(0, maxLength) + '...'
```

Comment:
This is adding `...` to every text in a button. Need to change this to put `...`'s only when the `maxLength` is crossed. Please fix